GETTING_STARTED.md

```diff
@@ -532,7 +532,8 @@ end
 This allows us to do:
 
 ```
-create(:comment)
+# Would be created for :photo by default since we have `for_photo` after :comment factory definition
```

Comment:
I think it's good to have some notes considering I'm not the only one who took some time to understand why `for_video` is missing in the definition (ref: https://github.com/thoughtbot/factory_bot/issues/1226#issuecomment-470272866)
I'm open to reverting this change if needed.

Reply:
Could we add this comment inline on line 519? I think we could save some words that way:

```rb
factory :comment do
  for_photo # default to the :for_photo trait if none is specified
```

Reply:
that's way better!